lib/unitarize_links_quda.cu

```diff
@@ -305,7 +305,7 @@ namespace{
     int num_failures = 0;
     Matrix<complex<double>,3> inlink, outlink;
       
-    for (int i=0; i<infield.Volume(); ++i){
+    for (unsigned int i=0; i<infield.Volume(); ++i){
```

Comment:
I feel like we've addressed this with `size_t` in other places. @maddyscientist what's the good word on this?

Reply:
No need for more than 32 bits here unless we care about 256^4 volume per process.  Certainly no harm in being future proof....

Reply:
I think the `unsigned` came in to silence a warning.